pywren/executor.py

```diff
@@ -135,9 +126,8 @@ def invoke_with_keys(self, s3_func_key, s3_data_key, s3_output_key,
 
         host_job_meta.update(arg_dict)
 
-        fut = ResponseFuture(call_id, callset_id, host_job_meta,
-                             self.s3_bucket, self.s3_prefix,
-                             self.aws_region)
+        storage_path = storage_utils.get_storage_path(self.storage_config)
```

Comment:
nit: This can probably be done at the beginning where we construct storage_config ?